javascript/javascript3/week1/ weatherApp.css

```diff
@@ -0,0 +1,42 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+  
+  body{
+    background-color: #D5DBDB ;
+    text-align: center;
+    justify-content: center;
+    font-family: cursive;
+    font-style: italic;
+    font-size: 24px;
+    font-weight: 300;
+  }
+  
+  h1{
+      text-shadow: 2px 2px #5499C7;
+      font-size: 50px;
+    }
+  input{
+  text-align: center;
+  border: 1px solid gray;
+  width: 200px;
+  height: 30px;
+  }
+  button{
+    border: 2px solid gray;
```

Comment:
Try to avoid styling the elements directly, Give them a className or Id, and then style them. It will be more helpful further.